### ISSUE TITLE
Lack of validation data

### ISSUE THREAD

**Issue by weihua916** [gusye1234/LightGCN-PyTorch#13]
Hi! In the paper, you mentioned, "The early stopping and validation strategies are the same as NGCF". However, I cannot find any validation set in the provided code. Am I missing anything? Thanks!

**Comment by gusye1234** [comment]
Sorry for the late response. 
We didn't write a Train-Validation-Test pipeline in our code. In fact, the validation metrics is the same as the testing, despite the difference in data. A possible way for you is to manually split a validation set from the training set, and then use it as the test data.
We are sorry about this, but the pytorch version of LightGCN is not the one we used during the experiments. This version is mainly filling vacancies and facilitating users. Maybe you can refer more details in the tensorflow version of code.
